src/common/components/layouts/pages/inspect/tree/Tree.js

```diff
@@ -83,6 +84,7 @@ function Tree({ ruleSummaries, selectedCheck, setSelectedCheck, errorsMap, profi
                 error => error.checkIndex === checkIndex && error.ruleIndex === ruleIndex
             );
             setSelectedCheck(sortedCheckIndex);
+            setTimeout(() => scrollToActiveBbox(), 100);
```

Comment:
There is no need to scroll to active bbox each time as it's done automatically by `verapdf-js-viewer` when selected index is changed. Call this method like that:
```
if (sortedCheckIndex === selectedCheck) {
  scrollToActiveBbox();
}
```

PS. Add `selectedCheck` to the list of dependencies.